src/utils/settings.ts

```diff
@@ -0,0 +1,39 @@
+import { DEFAULT_GAS_PRICE } from 'config'
+
+export interface UserSettings {
+  gasPrice: string
+}
+
+export const VERSION = 1
+
+export const GAS_SETTINGS = {
+  default: DEFAULT_GAS_PRICE, // 5
+  fast: '10000000000', // 10
+  reallyfast: '15000000000', // 15
```

Comment:
These are just initial values. I am sure they will be modified once we flesh out the requirements.